### DISCUSSION TITLE
Not able to set `www` domain on top of apex domain

### DISCUSSION THREAD

**Discussion by njkevlani** [community/community#51008] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey, I followed the guide from [Configuring an apex domain and the www subdomain variant](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain-and-the-www-subdomain-variant).

My domain is `njkevlani.dev`, which is working as expected, but `www.njkevlani.dev` is not failing to load with `ERR_CERT_COMMON_NAME_INVALID` error.

I'm a bit confused about how to debug this. Does anyone has some idea about this?

---

### Screenshot of the error

![image](https://user-images.githubusercontent.com/15014949/227720471-f1350985-ab56-48a3-bbe9-42d51fec72a4.png)

### The output of `dig`
```
$ dig www.njkevlani.dev +nostats +nocomments +nocmd
www.njkevlani.dev.      60      IN      CNAME   njkevlani.github.io.
njkevlani.github.io.    2143    IN      A       185.199.111.153
njkevlani.github.io.    2143    IN      A       185.199.108.153
njkevlani.github.io.    2143    IN      A       185.199.109.153
njkevlani.github.io.    2143    IN      A       185.199.110.153
$ dig njkevlani.dev +nostats +nocomments +nocmd
njkevlani.dev.          60      IN      A       185.199.109.153
njkevlani.dev.          60      IN      A       185.199.108.153
njkevlani.dev.          60      IN      A       185.199.110.153
njkevlani.dev.          60      IN      A       185.199.111.153
```
### Settings on my domain
![image](https://user-images.githubusercontent.com/15014949/227720572-da9b897c-9d5b-4cb2-8029-33ae565a532d.png)

---

Apart from that, I checked https://github.com/orgs/community/discussions/21837, which was exact same issue as mine, but it does not have an answer.

**Comment by njkevlani** [comment, marked as answer]
Any idea about this @yoannchaudet?

Edit: This was resolved after this: https://github.com/orgs/community/discussions/51008#discussioncomment-5482919.

**Reply by yoannchaudet** [reply]
👋 Not sure why your ALT certificate did not provision properly. This can happen if you don't visit again the settings Pages because we do an explicit check there.

In any case, I deleted your certificate and got a new one issued that covers both domains. Give it 10 minutes from this message to propagate through the CDN and you should be all set.

**Reply by njkevlani** [reply]
Hey,

Thanks, @yoannchaudet!! It worked :)

**Reply by Lifehack25** [reply]
Hey @yoannchaudet I have the same problem, my domain is drinkly.app, but the subdomain "www.drinkly.app" is not working, I had some problems with the SSL with GoDaddy but fixed it. maybe that has something to do with it. but I have talked with them and they are saying that it is not working because of something on your part. Can you do the same thing for me?

I have set up everything right with GoDaddy.

**Reply by yoannchaudet** [reply]
@Lifehack25 Check https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain-and-the-www-subdomain-variant. We prefer `www.drinkly.app` to be pointed to `<username>.github.io` instead of `drinkly.app`. Try making this change.

**Reply by Lifehack25** [reply]
@yoannchaudet tried switching that and have waited for over a hour, it still doesn't work.

**Reply by yoannchaudet** [reply]
Your TTL on the record was 1 hour so you will need to wait more than that. It is cached at different levels by the magic of DNS:

<img width="680" alt="image" src="https://github.com/community/community/assets/14911070/72e5c118-df1f-4a99-96fd-eee62bfa989a">

**Reply by Lifehack25** [reply]
Hey @yoannchaudet waited 20 hours now, it still doesn't work.

**Reply by yoannchaudet** [reply]
Confirmed the configuration looks great now. I forced re-issue a certificate for you. It includes the `www` variant. Give it 10 minutes to propagate through the CDN.

**Reply by Lifehack25** [reply]
Great, now it's working, thanks!
